docs/extras/updating.md

```diff
@@ -4,12 +4,17 @@ This page documents how you can keep your system up-to-date.
 
 After following our guide, your system will consist of three core elements that can be updated. Atmosphere, Hekate and your system firmware.
 
-### **Updating Atmosphere**
+## Updating Atmosphere
```

Comment:
Unrelated to this PR, but we should probably expand this page to feature common errors, and additional steps users may have to take (like updating/removing sysmodules).

May also be worth splitting this page off in 2 sepearte pages, one to update the firmware, one to update cfw.